skorch/callbacks/logging.py

```diff
@@ -728,3 +728,117 @@ def on_epoch_end(self, net, **kwargs):
     def on_train_end(self, net, **kwargs):
         if self.close_after_train:
             self.writer.close()
+
+
+class SacredLogger(Callback):
+    """Logs results from history to Sacred.
+
+    Sacred is a tool to help you configure, organize, log and reproduce
+    experiments. Developed at IDSIA. See https://github.com/IDSIA/sacred.
+
+    Use this callback to automatically log all interesting values from
+    your net's history to Sacred.
+
+    If you want to log additional information, you can simply add it to
+    ``History``. See the documentation on ``Callbacks``, and ``Scoring`` for
+    more information. Alternatively you can subclass this callback and extend
+    the ``on_*`` methods.
+
+    To use this logger, you first have to install Sacred:
+
+    >>> pip install sacred
```

Comment:
```suggestion
    $ pip install sacred
```
Since this is not run in a Python prompt.